cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -175,20 +185,26 @@ def test_provision_cloud_init_payload(appliance, request, setup_provider, provid
 
     # Provision VM
     collection = appliance.provider_based_collection(provider)
-    instance = collection.create(vm_name, provider, form_values=inst_args)
-    request.addfinalizer(instance.cleanup_on_provider)
+    instance: cfme.common.vm.VM = collection.create(vm_name, provider, form_values=inst_args)
+    request.addfinalizer(lambda: instance.cleanup_on_provider(handle_cleanup_exception=False))
+
     provision_request = provider.appliance.collections.requests.instantiate(vm_name,
                                                                             partial_check=True)
     check_all_tabs(provision_request, provider)
     provision_request.wait_for_request()
 
-    connect_ip = wait_for(find_global_ipv6, func_args=[instance], num_sec=600, delay=20).out
-    logger.info(f'Connect IP: {connect_ip}')
+    wait_for(lambda: ci_payload['ip_address'] in instance.mgmt.all_ips)
+
+    vmip, = (ip for ip in instance.mgmt.all_ips if ip.startswith('2620'))
+    jump_host_config = conf.cfme_data['management_systems'][provider.key]['hosts'][0]
+    jump_host_creds = conf.credentials[jump_host_config['credentials']['default']]
+
+    with closing(SSHClient(hostname=jump_host_config['name'],
```

Comment:
Why are these calls wrapped in closing? The SSHClient context manager should be handling closing the tunnel on exit. 

If its necessary for some reason, please include the description of why as a comment block here.

Reply:
done